tests/test_animal_shogi.py

```diff
@@ -223,9 +223,26 @@ def test_repetition():
     assert (state.rewards == 0).all()
 
 
-
 def test_api():
     import pgx
     env = pgx.make("animal_shogi")
     pgx.api_test(env, 3, use_key=False)
     pgx.api_test(env, 3, use_key=True)
+
+
+def test_buggy_samples():
+    # 1209

```

Comment:
```suggestion
    # https://github.com/sotetsuk/pgx/pull/1209
```

makes easy to jump from IDEs